tests/nested/manual/core20-factory-reset-install-device-hook/install-device

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# install the files the same way the specific device does this - note we don't
+# create the directories since that requires extra permissions we don't have 
+# here, system-files only allows us access to the specific files
+
+# add a udev rule if executed from factory-reset
+if snapctl system-mode | grep "factory-reset"; then
+    cat << EOF > /run/mnt/ubuntu-data/system-data/_writable_defaults/etc/udev/rules.d/from-factory-reset.conf
+# this udev rule should only appear after factory-reset mode
+EOF
```

Comment:
you could use a simple `echo` instead of cat, since it's only one line. But this is OK too.

Maybe defining `WRITABLE_DIR="/run/mnt/ubuntu-data/system-data/_writable_defaults"` at the top of the script would reduce the risk of typos.

Reply:
Nice suggestion, I'll leave the cat as it makes no difference, but adding the `WRITABLE` macro!